README.md

```diff
@@ -12,126 +10,13 @@ We demonstrate that the available data sets can directly profit from the most re
-            MapToDataEditor(),
-            # replace NaN with 0
-            NanEditor(),
-            # normalize data to [-1, 1]
-            NormalizeEditor(norm),
-            # change data to channel first format
-            ReshapeEditor((1, resolution, resolution))]
-        super().__init__(path, editors=editors, ext=ext, **kwargs)
-```
-
-### Create custom trainings
-
-The trainer class implements a basic training function, that requires the workspace directory and the low- and high-quality dataset.
-An optional validation data set can be specified to verify the model results (plots and parallel validation).
-
-In the example bellow we translate SOHO observations to SDO quality. Both instruments have 5 channels (input_dim_a/b=5). We increase the resolution by a factor 2 (upsampling=1). 
-We expect mostly instrumental characteristics that cause degradations and set the diversity factor to 0 (lambda_diversity=0).
-For the training we specify the SDO and SOHO datasets, where we use a fixed resolution of 1024 pix for SOHO and consequently 2048 pix for SDO (1 times upsampling).
-The training is performed with images patches, that we sample from the full-disk observations. According to our GPU memory we select a patch size of 128 pix for SOHO (256 pix for SDO).
-We apply a temporal separation of our dataset, where we use the months 1-10 for training and 11-12 for validation.
-
-Images are automatically saved during training, but note that they will only provide information about the qulaity of the 
-translation when the InstanceNormalization weights are fixed (after 100 000 iterations).
-The use of learned parameters of the InstanceNormalization is required for the training with image patches.
-
-```python
-from iti.data.dataset import SDODataset, SOHODataset
-from iti.train.model import DiscriminatorMode
-from iti.trainer import Trainer
-
-base_dir = ""
-sdo_data_path = ""
-soho_data_path = ""
-# Init model
-trainer = Trainer(input_dim_a=5, input_dim_b=5, upsampling=1, discriminator_mode=DiscriminatorMode.CHANNELS,
-                  lambda_diversity=0)
-
-# Init training datasets
-sdo_train = SDODataset(sdo_data_path,
-                       resolution=2048, patch_shape=(256, 256),
-                       months=list(range(11)))
-soho_train = SOHODataset(soho_data_path,
-                         resolution=1024, patch_shape=(128, 128),
-                         months=list(range(11)))
-# Init validation/plotting datasets
-sdo_valid = SDODataset(sdo_data_path,
-                       resolution=2048, patch_shape=(256, 256),
-                       months=[11, 12], limit=100)
-soho_valid = SOHODataset(soho_data_path,
-                         resolution=1024, patch_shape=(128, 128),
-                         months=[11, 12], limit=100)
+## Usage
+To get started with the ITI tool we recommend to play around with the [example](https://iti-documentation.readthedocs.io/en/latest/generated/gallery/index.html) notebooks.
```

Comment:
play around is bad wording.
--> We provide an example gallery that can be used as a starting point.

Reply:
added example notebook to examples folder.

Reply:
changed wording.